packages/story-editor/src/karma/fixture/containers/designPanel/textStyle.js

```diff
@@ -103,7 +104,59 @@ export class TextStyle extends AbstractPanel {
     return this.getByRole('option', { name: /fill/i });
   }
 
+  get addStyle() {
+    return this.getByRole('button', { name: /Add style/ });
+  }
+
+  get applyStyle() {
+    return this.getByRole('button', { name: /Apply style/ });
+  }
+
+  get presets() {
+    return this.getAllByRole('button', { name: /Apply style/ });
+  }
+
+  get moreStyles() {
+    return this.getByRole('button', { name: /More styles/ });
+  }
+
+  get styleManager() {
+    return this._get(
+      this.getByRoleIn(this.node.ownerDocument, 'dialog', {
+        name: /Style presets manager/,
+      }),
+      'styleManager',
+      StyleManager
+    );
+  }
+
   // @todo: add remaining input options:
   // * font family and size
   // * justify toggles
 }
+
+class StyleManager extends Container {
+  constructor(node, path) {
+    super(node, path);
+  }
+
+  get apply() {
+    return this.getByRole('button', { name: /Apply style/ });
+  }
+
+  get presets() {
+    return this.getAllByRole('button', { name: /Apply style/ });
+  }
```

Comment:
These can't both exist, or? Won't `getByRole` throw if there's more than one? Or do you make sure to only access `.apply` if there's **only one** preset?

Reply:
Oh, I see it's the same in the main panel - and was in the old panel as well. I guess you just have to be careful which one you call.